app/controllers/posts_controller.rb

```diff
@@ -0,0 +1,80 @@
+class PostsController < ApplicationController
+  before_action :set_post, only: %i[show edit update destroy]
+  before_action :authenticate_member!, except: %i[index show]
+
+  # GET /posts
+  # GET /posts.json
+  def index
+    @posts = Post.all.order('created_at DESC')
+    @members = Member.all
+  end
+
+  # GET /posts/1
+  # GET /posts/1.json
+  def show; end
+
+  # GET /posts/new
+  def new
+    @post = current_member.posts.build
+  end
```

Comment:
* [ ] it should initialize an empty post `@post = Post.new` ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/authentication/posts_new_requirement.png) from [Odin](https://www.theodinproject.com/courses/ruby-on-rails/lessons/authentication))